crates/sui-core/src/consensus_manager/mysticeti_manager.rs

```diff
@@ -110,10 +110,10 @@ impl ConsensusManagerTrait for MysticetiManager {
 
         let name: AuthorityName = self.keypair.public().into();
 
+        let sui_committee = system_state.get_sui_committee();
```

Comment:
There shouldn't be a difference (in theory & in practice) - nothing wrong reading from epoch store - but since we are getting the mysticeti committee via the system state then I've considered more straightforward to do the same for the SUI committee as well so we don't read relevant information from two different sources.